Resources/Locale/en-US/species/shadowkin.ftl

```diff
@@ -0,0 +1,15 @@
+shadowkin-power-examined-other = {CAPITALIZE(SUBJECT($target))} seems to be {$powerType}.
+shadowkin-power-examined-self = You have {$power}/{$powerMax} energy, you are {$powerType}.
```

Comment:
So while this is something only the controlling player can see, it looks weird in the inspect menu to switch between 3rd and 2nd person.

I personally think it should switch to 3rd person, but I'm not sure, and I wanted to bring this up for discussion.